tests/main/listing/task.yaml

```diff
@@ -20,19 +20,19 @@ execute: |
     #shellcheck disable=SC2166
     if [ "$SPREAD_BACKEND" = "linode" -o "$SPREAD_BACKEND" = "google" -o "$SPREAD_BACKEND" == "qemu" ] && [ "$SPREAD_SYSTEM" = "ubuntu-core-16-64" ]; then
         echo "With customized images the core snap is sideloaded"
-        expected='^core .* [0-9]{2}-[0-9.]+(~[a-z0-9]+)?(\+git[0-9]+\.[0-9a-f]+)? +x[0-9]+ +- +- +core *$'
+        expected="^core .* [0-9]{2}-[0-9.]+(~[a-z0-9]+)?(\+git[0-9]+\.[0-9a-f]+)? +x[0-9]+ +- +- +core *$"
```

Comment:
What's the point of changing the string to double quotes? Is this a shellcheck-driven change?

Reply:
@zyga Thanks for reviewing it
You are right, no need to change that but I did it just to use the same type of quoting in all the lines. 
I am reverting that.